Procurement/ViewModel/ItemHoverViewModel.cs

```diff
@@ -104,6 +118,18 @@ public ItemHoverViewModel(Item item)
                 ExperienceDenominator = gem.ExperienceDenominator;
             }
 
+            if (IsIncubatorProgressVisible)
+            {
+                IncubatorNumerator = item.IncubatedDetails.Progress;
+                IncubatorDenominator = item.IncubatedDetails.Total;
+                Incubating = $"Incubating {item.IncubatedDetails.Name}";
+                IncubationLevel =  $"Level {item.IncubatedDetails.Level}+ Monster Kills";
+                if (Item.IncubatedDetails.Total > 0 )
```

Comment:
No space before the `)`.

Reply:
Fixed, we should get style cop for these sorts of nits.